packages/webpack.browser.config.js

```diff
@@ -47,6 +47,7 @@ const webConfig = {
             fs: false,
             crypto: require.resolve('crypto-browserify'),
             'fs-extra': false,
+            perf_hooks: false, // should be using globalThis.performance instead
```

Comment:
👍

Reply:
Yup created a ticket to start linting these dependencies